### ISSUE TITLE
helm chart does not add acl for ingressclass in clusterrole (v1.6.1)

### ISSUE THREAD

**Issue by havardelnan** [vmware/load-balancer-and-ingress-services-for-kubernetes#747]
### Describe the bug

Ako pod is unable to read ingressclasses in v1.6.1 resulting in repeating:

E0106 07:29:13.478723       1 reflector.go:138] github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/internal/k8s/controller.go:1087: Failed to watch *v1.IngressClass: failed to list *v1.IngressClass: ingressclasses.networking.k8s.io is forbidden: User "system:serviceaccount:avi-system:ako-sa" cannot list resource "ingressclasses" in API group "networking.k8s.io" at the cluster scope

### Reproduction steps

```bash
1. helm install ako v 1.6.1
2. get logs of ako pod


Adding ingressclass manualy in the clusterrole fixes the error and allows ako to continue.
```

### Expected behavior

That the ako-sa serviceaccount are able to read ingressclasses using the provided clusterrole.

### Additional context

Going back to 1.5.1 works.

**Comment by chauhanshubham** [comment]
The 1.6.1 cluster role must have the ingressclasses rule already
https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/0a3b1fa66aeaaf77c32599d0985957b7d83f395a/helm/ako/templates/clusterrole.yaml#L20

**Comment by chauhanshubham** [comment]
Another reason that I can see for this could be that the kubernetes version you use is prior to v1.19.
AKO added support for IngressClass only after v1 was released in v1.19 k8s. Therefore AKO programs the cluster role with `ingressclasses` in it, only in presence of the v1 API and not v1beta1. 
We do not support IngressClass in v1beta1 and would recommend the IngressClass usage on from v1.19 kubernetes release.
